src/messages/textSize.json

```diff
@@ -1,5 +1,7 @@
 {
     "rangeInstructions": "Adjust the text to a size you like best.",
     "increaseLabel": "larger",
-    "decreaseLabel": "smaller"
+    "increaseTooltip": "Select to increase size",
+    "decreaseLabel": "smaller",
+    "decreaseTooltip": "Select to decrease size"
```

Comment:
Because the tooltip is recreated every time a button is clicked, for now we are going to leave the tooltip for increase and decrease as "larger" and "smaller"